From d40882afb05ce56685a882e1167b8ec1613d478c Mon Sep 17 00:00:00 2001
From: Hojjat <mobin@mobintestserver.ir>
Date: Tue, 13 Dec 2022 15:23:18 +0330
Subject: [PATCH] meson: Fix `loadtests` on meson builds (#133)

Signed-off-by: Mobin 'hojjat' Aydinfar <mobin@mobintestserver.ir>
---
 src/tests/meson.build | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/src/tests/meson.build b/src/tests/meson.build
index 76cc3b7b..dec60b25 100644
--- a/src/tests/meson.build
+++ b/src/tests/meson.build
@@ -25,6 +25,16 @@ configure_file(
     configuration : mconfig_data
 )
 
+## Prepare loadtests
+run_command([
+    'cp',
+    '-r',
+    meson.current_source_dir() + 'test-services',
+    meson.current_build_dir()
+    ],
+    check : true
+)
+
 ## Outputs
 # Unit tests
 tests_exec = executable(
@@ -57,6 +67,6 @@ envtests_exec = executable(
 )
 test('tests', tests_exec)
 test('proctests', proctests_exec)
-test('loadtests', loadtests_exec, should_fail : true)
+test('loadtests', loadtests_exec, workdir : meson.current_build_dir())
 test('envtests', envtests_exec)
 subdir('cptests/')
